README.md

```diff
@@ -90,3 +90,16 @@ For Payment Hub usecases, check this [documentation](https://mifos.gitbook.io/do
 ## Note
 
 The UI design is currently being revamped. New design can be found [here](https://docs.google.com/presentation/d/1yFR19vGlKW-amxzGms8TgPzd1jWkrALPFcaC85EyYpw/edit#slide=id.g6c6ccd991d_0_42)
+
+## Join Us on Slack
```

Comment:
Also add in this table https://github.com/openMF/mifos-mobile/blob/development/README.md?plain=1#L9

Reply:
done